### ISSUE TITLE
No drawing of gestures on tomshardware.com

### ISSUE THREAD

**Issue by ryanbr** [Robbendebiene/Gesturefy#41]
http://www.tomshardware.com/ (and its internal links) doesn't show the drawing of the gesture. It works still,  just doesn't show the gesture

**Comment by Robbendebiene** [comment]
Which version are you using? Because this may be fixed in the new version 1.1.8.

**Comment by ryanbr** [comment]
Using 1.1.8 on todays nightly

**Comment by Robbendebiene** [comment]
Hm strange its working fine for me on the stable release and version 56. Are there any errors in the console after you visit the page/try to make a gesture? (you can open the console by pressing: ctrl + shift + J)

**Comment by ryanbr** [comment]
Using builds from https://treeherder.mozilla.org/#/jobs?repo=mozilla-central&filter-searchStr=windows%202012%20opt  

Windows 2012 x64 pgo to be exact, I can't see any console errors being generated (is there -dev version with any additional flags enabled?)

**Comment by Robbendebiene** [comment]
So you are using firefox nightly 57?

>  (is there -dev version with any additional flags enabled?)

I don't think so.

**Comment by Robbendebiene** [comment]
@ryanbr Does this still happen to you in Firefox 57?

**Comment by ryanbr** [comment]
Installed beta 4, gestures aren't showing on tomshardware. (other sites seem fine)

**Comment by ryanbr** [comment]
Okay it seems if you're using an Adblock/UbO, it'll cause the issue. Specifically these 2 filters (used for Anti-adblock)

`
tomshardware.co.uk,tomshardware.com##[style*="animation:"]
tomshardware.co.uk,tomshardware.com##[style*="transition:"]
`

**Comment by Robbendebiene** [comment]
Yeah you are right. The selection of the filter should be much more specific in my eyes.
